bundle/manifests/lightspeed-operator.clusterserviceversion.yaml

```diff
@@ -324,9 +312,11 @@ spec:
             spec:
               containers:
               - args:
-                - --health-probe-bind-address=:8081
-                - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=:8080
```

Comment:
if the metrics endpoint is now directly TLS encrypted, we should move it to port 8443 to reflect that, 8080 should be used for unencrypted endpoints.